### PR TITLE
Add types

### PR DESCRIPTION
**Pull request by sfc-gh-pkommini**
- Fix files to include types
- Add the `index.d.ts` with the declaration
- Fix linting issues using `prettier`
- Increment version

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "snowsql-formatter",
-  "version": "1.0.0",
+  "version": "1.1.0",
```
Comment (sfc-gh-afedorov) [inline package.json:3]: did you mean for this to match what's in `package-lock.json`?

**Review by sfc-gh-afedorov — APPROVED** [review]
minor fixes, lgtm otherwise

File: package.json (hunk 1 above)
Comment (sfc-gh-pkommini) [reply package.json:3]: Oh meant to delete that file. Do you think it makes sense to include that in the repo? This [doc](https://github.com/npm/cli/blob/release-6.14.7/docs/content/configuring-npm/package-lock-json.md) seems to suggest that we should push it to version control.

Comment (sfc-gh-afedorov) [reply package.json:3]: I think so -- that "integrity" piece is what allows us to have reproducible builds, i.e. making sure that we have an audit trail of which code was included at which point

Comment (sfc-gh-pkommini) [reply package.json:3]: Alright. Will include it with the correct version.

File: types/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+// Type definitions taken from sql-formatter 2.3
+// Definitions by: Jake Boone <https://github.com/jakeboone02>
```
[the remark this answers is not in the document]
Comment (sfc-gh-pkommini) [reply types/index.d.ts:2]: The entire library here of types is MIT licensed. https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/LICENSE and so would that include the subdirectories? None of the libraries in that repo have a license, probably due to the fact that the definitelytyped repo is globally licensed? Do we need to include legal for review?

**Pull request merged**